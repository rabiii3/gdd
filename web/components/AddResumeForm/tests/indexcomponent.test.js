import React from 'react';
import { shallow } from 'enzyme';
import { AddForm } from '../';

describe('web | components | modalandform | componentindex', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        setModal: () => {},
        isModalOpen: false,
        classes: {},
        file: [],
      };
      const wrapper = shallow(<AddForm {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
