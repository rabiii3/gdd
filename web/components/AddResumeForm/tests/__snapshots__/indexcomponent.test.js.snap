// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | modalandform | componentindex render should match snapshot 1`] = `
<WithStyles(Modal)
  onClose={[Function]}
  open={false}
>
  <div
    style={
      Object {
        "left": "50%",
        "top": "50%",
        "transform": "translate(-50%, -50%)",
      }
    }
  >
    <WithStyles(Typography)
      id="simple-modal-description"
      variant="subtitle1"
    >
      <Connect(WithStyles(ResumeForm))
        file={Array []}
        setModal={[Function]}
      />
    </WithStyles(Typography)>
  </div>
</WithStyles(Modal)>
`;
